### ISSUE TITLE
Remove tagline from Site template

### ISSUE THREAD

**Issue by teresasilva71** [ucsc/ucsc-2022#111]
![tagline](https://user-images.githubusercontent.com/7795841/182993628-d5bbccf6-e868-4422-b647-89d9ce3d758c.png)
Please remove the tagline from the site template. I couldn't see a way to do this.
It looks like it might be the template description that's appearing. Tom says it also appears when hovering over the favicon in the browser tab.

**Comment by Herm71** [comment]
Hi @teresasilva71 Generally all WordPress sites inject a default tagline (used to be called "description") on a fresh install. It is one of the defaults that typically get changed or edited upon site creation. It's the second field on the **Settings->General** page, right below the site's title, which is Diplomas. I can discuss removing it from the site template with Rob but I believe it should be the the Site Manager's responsibility to remove or replace it when they fill out the other info on that page, such as admin email, etc (Kumar, in this case).

In any case, I went in and removed this from Diplomas. 
![wordpress-tagline](https://user-images.githubusercontent.com/1000543/183111601-b8f66a44-4a1b-400d-9fcf-d048486f2039.png)

**Comment by knice** [comment]
"_it should be the the Site Manager's responsibility to remove or replace it when they fill out the other info on that page_"

@teresasilva71, @Herm71 is correct. This is something the site manager must manage on their site. I was planning to talk about in the orientations because the text in this field is added to the HTML meta description tag. The description tag text is commonly (but not always) shown beneath the title in search engine results.

Bottom line: it must be filled out and customized for each site. I can change the text to something like "Go to Settings > General to customize your site's tagline" or something. But we can't leave that field empty.

![screenshot-2022-08-05-091929](https://user-images.githubusercontent.com/36719/183119606-1bf34293-3c89-4b99-a3b3-56ecb6b08f9c.jpg)

**Comment by knice** [comment]
We updated the tagline to read: _This tagline is shown in search engine results. Change your site's tagline in Settings > General._
